docs/documentation/server_admin/topics/authentication/authentication-sessions.adoc

```diff
@@ -13,13 +13,13 @@ The authentication session usually expires after 30 minutes by default. The exac
 
 As described in the previous section, a situation can involve a user who is trying to authenticate to the {project_name} server from multiple tabs of a single browser. However, when that user authenticates in one browser tab,
 the other browser tabs will automatically restart the authentication. This authentication occurs due to the small javascript available on the {project_name} login pages. The restart will typically
-authenticate the user in other browser tabs and redirect to clients because there is an SSO session now due to the fact that the user just successfully authenticated in first browser tab. 
+authenticate the user in other browser tabs and redirect to clients because there is an SSO session now due to the fact that the user just successfully authenticated in first browser tab.
 Some rare exceptions exist when a user is not automatically authenticated in other browser tabs, such as for instance when using an OIDC parameter _prompt=login_ or <<_step-up-flow, step-up authentication>>  requesting a stronger
 authentication factor than the currently authenticated factor.
 
 In some rare cases, it can happen that after authentication in the first browser tab, other browser tabs are not able to restart authentication because the authentication session is already
 expired. In this case, the particular browser tab will redirect the error about the expired authentication session back to the client in a protocol specific way. For more details, see the corresponding sections
-of  link:{adapterguide_link}#_oidc-errors[OIDC documentation] and  link:{adapterguide_link}#_saml-errors[SAML documentation]. When the client application receives such an error, it can immediately resubmit the OIDC/SAML authentication request to {project_name} as
+of  *OIDC documentation* in the link:https://www.keycloak.org/guides#securing-apps[securing apps] section and  link:{adapterguide_link}#_saml-errors[SAML documentation]. When the client application receives such an error, it can immediately resubmit the OIDC/SAML authentication request to {project_name} as
```

Comment:
Can we create some macro/attribute for the `https://www.keycloak.org/guides#securing-apps` ? As for the product documentation, this link will be different place.

I can see there are already some like for example https://github.com/keycloak/keycloak/blob/main/docs/documentation/topics/templates/document-attributes.adoc#L75 or https://github.com/keycloak/keycloak/blob/main/docs/documentation/topics/templates/document-attributes.adoc#L78 , so hopefully something like can be used for "Securing apps guides" as well?

I think this may apply to SAML as well as the link to "SAML documentation" on the same line is probably invalid (as AFAIK @rmartinc already moved that one to "Guides")

Reply:
@mposolda thanks for the review! added an attribute for  https://www.keycloak.org/guides#securing-apps in a separate commit. The SAML errors page at this link `link:{adapterguide_link}#_saml-errors` was not moved to guides @rmartinc